src/helpers/referal.js

```diff
@@ -86,7 +88,38 @@ __private.list = function (filter, cb) {
     }).catch((err) => {
         library.logger.error(err.stack);
         return setImmediate(cb, 'Rewards#list error');
+    }); */
+
+    /** Resolved Count Issue on Response with Updated Code */
+
+    library.db.query(sql.countList({
```

Comment:
put count in query USE WINDOW FUNCTIONS `count(1) over ()`

Reply:
Okay, will do it as it was previous logic that's why we didn't change.